index.qmd

```diff
@@ -1,40 +1,40 @@
+---
+description-meta: |
+  This book will teach you how to work with geographic data in Python.
+  It's a short and practical open source book in which you'll
+  develop and use geocomputation to solve practical
+  problems and lay the foundations for advanced geospatial applications.
+---
 
-# Introduction
+## Welcome {.unnumbered}
 
-<!-- This site contains ideas, code and an outline of a yet-to-be written book on *Geocomputation with Python*. -->
+This is the online home of *Geocomputation with Python*, an book on reproducible geographic data analysis modeling with open source software.
 
-## Motivations
+Inspired by the Free and Open Source Software for Geospatial ([FOSS4G](https://foss4g.org/)) movement, the code and prose underlying this book are open, ensuring that the content is reproducible, transparent, and accessible.
+The code is hosted on [GitHub](https://github.com/geocompr/py), allowing anyone to interact with the project by opening issues or contributing new content and typo fixes for the benefit of everyone.
 
-**Geocomputation with Python** (*geocompy*), is motivated by the need for an introductory yet rigorous and maintained resource on working with geographic data in Python.
-A unique feature of the book is that it that demonstrates code for working with *both vector and raster* geographic data types.
-<!-- , and how to use them together. -->
-There are many resources on Python packages for geographic research and various applications but, to the best of our knowledge, no other resource brings together the following features into a single home:
+Its current 'build status' as follows:
```

Comment:
@Robinlovelace maybe it would be good to have it as a conditional content -- https://quarto.org/docs/authoring/conditional.html?